src/bootupd.rs

```diff
@@ -489,6 +490,149 @@ pub(crate) fn client_run_validate() -> Result<()> {
     Ok(())
 }
 
+pub(crate) fn client_run_migrate() -> Result<()> {
+    // Used to condition execution of this unit at the systemd level
+    let stamp_file = "/boot/.bootupd-static-migration-complete";
+
+    // Did we already complete the migration?
+    let mut ostree_cmd = std::process::Command::new("ostree");
+    let result = ostree_cmd
+        .args([
+            "config",
+            "--repo=/sysroot/ostree/repo",
+            "get",
+            "sysroot.bootloader",
+        ])
+        .output()
+        .with_context(|| "failed to call ostree command. Not performing migration")?;
```

Comment:
contexts should be a single sentence fragment; this will appear strangely when rendered.
I'd just use:
`.context("Querying ostree sysroot.bootloader")` or so